### DISCUSSION TITLE
Tabs like collections of buffers

### DISCUSSION THREAD

**Discussion by Nosterx** [neovide/neovide#1811] [Q&A] [answered]
I used to open number of project at same time in different terminal tabs with terminal version of neovim. I there a way to achieve this with neovide?

**Comment by erlangparasu** [comment]
did you mean something like grouping buffers per tab? https://github.com/tiagovla/scope.nvim

**Comment by Nosterx** [comment]
Visually - yes, somethink like that. But in my imagination every tab should have different backend neovim process that will have it's own cwd and set of environment variables for plugins to work correctly. I dunno why I asked this question - for now it's clear for me that currently this is not possible with one instance of neovide.

**Reply by fredizzimo** [reply, marked as answer]
Neovim already support a different working directory per tab https://neovim.io/doc/user/editing.html#%3Atcd, that's how I personally work with one project per tab using https://github.com/ahmedkhalf/project.nvim. 

There are also tab-local variables https://neovim.io/doc/user/builtin.html#gettabvar(), so it's possible to make tab aware plugins. Or even manually in your config, save the variables as tab variables, then setup an autocommand to set it globally when you switch tabs.

**Reply by Nosterx** [reply]
Thank you for your detailed response!  :+1:
